website/docs/sql-function-glossary/sql-aggregate-functions/sql-sum.md

```diff
@@ -0,0 +1,60 @@
+---
+id: sql-sum
+title: SQL SUM 
+description: The SUM aggregate function allows you to calculate the sum of a numeric column or across a set of rows for a column.
+slug: /sql-function-glossary/sql-sum
+---
+
+<head>
+    <title>Working with SQL SUM</title>
+</head>
+
+The SQL SUM function is handy and ever-present in data work. Let’s unpack what it is, how to use it, and why it's valuable.
+
+Jumping into it, the SUM aggregate function allows you to calculate the sum of a numeric column or across a set of rows for a column. Ultimately, the SUM function is incredibly useful for calculating meaningful business metrics, such as LTV, and creating key numeric fields in `fct_` and `dim_` models.
```

Comment:
Maybe spell out lifetime value (aka LTV) as people might not know what that means

Reply:
And do we have anything we can link to for fct + dim models throughout this glossary (maybe the dimensional modeling glossary post)?

Reply:
agree to both these comments @johnblust

Reply:
Done!